### ISSUE TITLE
Multipart max chunk size seems to be ~2GB

### ISSUE THREAD

**Issue by zhiwei-nu** [s3tools/s3cmd#953]
```
WARNING: Upload failed: /backup_2018-01-29.pgdump?partNumber=1&uploadId=AnLA52QDBz8P8LmzTDLyfgzanQtv93kGBjBIdmIbu1sxMnDUautbiNPitWvRlyZcSdXPEjE84qJL2hn.6itXDZMwsrzn4xgO2C5znN4O0t5hnOefpxQKt5f3JOjY6vX9aCYjeflz_dacG_ynuE9C_FQ2wjEB3WstLhfdwEpUPhk- (string longer than 2147483647 bytes)
```
Tried to upload using `5000 MB` for multipart chunk size and got this error. Upload now working using `2000 MB` for the chunk size.

**Comment by fviard** [comment]
@zhiwei-nu Did you try any size in between 2 and 5? Like 4900 MB?
Because the maximum size for a chunk on aws S3 is 5GB.

**Comment by zhiwei-nu** [comment]
@fviard i'll give it a quick try. I settled onto 2000MB because the error message seemed to suggest that: `(string longer than 2147483647 bytes)`

**Comment by zhiwei-nu** [comment]
@fviard Seems that it works properly, feel free to close it (or check the limit to be <= 5000 MB).

As a side note, I'm getting this:
```
WARNING: Upload failed: /testdump?partNumber=1&uploadId=pNS8A0LIU22EouFg1jsQUa8MQBHXpZWzignDI_i_0VhCl2fDDtZHLHDM.83P1vhr0FcgDdLTXZE_1rf06X32fLJQlzxq8hqLUIs4Y8Qhg0w1YUPtfyxANLVH5sb7YuiQnVGz474pKPPuK_95bYo7fXtE0x9SYlMPh3P3pceXRps- ([Errno 32] Broken pipe)
WARNING: Retrying on lower speed (throttle=0.00)
WARNING: Waiting 3 sec...
```

Although the upload works fine, I suspect there is duplicate work being done here - is it possible to avoid it?

**Comment by fviard** [comment]
@zhiwei-nu fyi your "retrying" issue should now be fixed with #1114 and will be shipped in the next release.
